includes/documents/class-wcpdf-bulk-document.php

```diff
@@ -41,6 +41,9 @@ public function __construct( $document_type, $order_ids = array() ) {
 		$this->type      = $document_type;
 		$this->order_ids = $order_ids;
 		$this->is_bulk   = true;
+		
+		// output formats (placed after parent construct to override the abstract default)
+		$this->output_formats = apply_filters( "wpo_wcpdf_{$this->type}_output_formats", [ 'pdf' ], $this );
```

Comment:
I just wanted to have consistency between all documents, even understanding that this is a different kind of document. Now because I have added the `is_enabled()` method, I'm using this property.